src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -295,3 +295,204 @@ and if they have a value depending on the event origin.
+Steps on how to create the App are present in the following description.
+
+The Notification's Third-Party script needs 2 IDs and a secret, that are retrieved from the Microsoft Entra admin center:
+
+- A *tenantId* -> The ID of your organisation
+- A *clientId* -> The ID of the registered App
+- A *secret* -> The "password", encoded using a ProActive tool, used to authenticate the app when querying the Graph API
+
+====== Connect to the Microsoft entra center
+
+- Connect to your Microsoft 365 account from https://account.microsoft.com/account/Account[here]
+- Go to "My Apps" from the left menu
+
+image::microsoft-my-apps-button.png[align="center"]
+
+- Access the Admin center by clicking on the Admin app
+
+image::microsoft-apps-dashboard-admin.png[align="center"]
+
+- Access the Microsoft Entra admin center by clicking on "Identity" in the "Administration centers" left menu
+//TODO get screenshot image::microsoft-apps-dashboard-Entra-admin-button.png[align="center"]
+- Click on the "App Registration" sub menu in the "Applications" left menu
+
+image::entra-admin-center.png[align="center"]
+
+====== OPTIONAL: Create an App
+
+ 1. Click on the "New Registration" button
+image::create-app-button.png[align="center"]
+ 2. Provide a name for the App. Here we use the name NotificationService.
+ 3. Leave the default Supported account type in single tenant
+ 4. Finally, click on the "Register" button
+image::microsoft-entra-register-app.png[align="center"]
+
+====== Copy the tenantId and clientID
+
+- If you just created an App, you land on the App overview.
+- If you use an existing App, click on "All applications" and click on your app
+
+Copy and save the tenantId and clientId
+
+image::microsoft-app-tenant-client-id.png[align="center"]
+
+====== Give the app the necessary API permissions
+
+The App needs to have the authorisation to Read Users and Send messages to channels as shown below
+
+image::microsoft-app-api-permission.png[align="center"]
+
+To add these permissions:
+
+- From the App overview, click on the API permission left menu
+- Click on the "Add Permission button"
+- Click on "Microsoft Graph"
+
+image::microsoft-app-graph-permission.png[align="center"]
+
+- Click on "Delegated Permission" (because the authorisation is delegated to users, not the app itself)
+- In the search bar, enter ChannelMessage.Send, select the permission then click on "Add permission"
+
+image::microsoft-add-permission.png[align="center"]
+
+- Do the same for the permission User.Read
+- Click on "Grant admin consent for ..."
+
+image::microsoft-app-grant-admin-consent.png[align="center"]
+
+====== Create a secret
+
+To authenticate itself, the app uses a secret. It must be created on this portal.
+
+WARNING: After the secret is created, it can be read and copied once. If lost, it is definitely lost and a new secret must be created, encoded, and replaced in the Third-Party script.
```

Comment:
to add: "Store the secret in a secure vault or encrypted file. Avoid sharing it in plain text to prevent unauthorized access."

Reply:
OK